projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -203,6 +208,31 @@ public static function plugin_activated( $plugin ) {
 		}
 	}
 
+	/**
+	 * Runs when a new comment is added.
+	 *
+	 * @param integer $comment_id The ID of the comment that was added.
+	 * @param bool    $comment_approved Whether or not the comment is approved.
+	 * @param array   $commentdata Comment data.
+	 */
+	public static function comment_added( $comment_id, $comment_approved, $commentdata ) {
+		if ( is_plugin_active( 'akismet/akismet.php' ) || self::is_conditional_recommendation_enabled( self::ANTI_SPAM_RECOMMENDATION ) ) {
```

Comment:
When adding a comment on the front end I'm getting a fatal error that `is_plugin_active()` isn't defined. 🤔 
Is it possible that function hasn't loaded yet at the point when this runs?
<img width="912" alt="Screen Shot 2022-04-01 at 1 36 37 PM" src="https://user-images.githubusercontent.com/690843/161337866-12a036b3-3a98-48a7-8d7e-6afd261d1b27.png">

Reply:
Thanks for catching that! I think `is_plugin_active` only loads in the admin context 🤔 I wonder why this didn't fail for me! Leaving this codex page here for my future reference :) 
https://developer.wordpress.org/reference/functions/is_plugin_active/